enhancements/helm3/namespace-scoped-helm-repository-crd.md

```diff
@@ -0,0 +1,158 @@
+---
+title: namespace-scoped-helm-repository-crd
+authors:
+  - "@zonggen"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-11-03
+last-updated: 2021-11-03
+status: implementable
+---
+
+# Namespace-scoped Helm Repository CRD
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Right now we support [`cluster-scoped` Helm repository CRD](https://github.com/openshift/api/blob/master/helm/v1beta1/0000_10-helm-chart-repository.crd.yaml#L17). This requires admin-level permissions when developers want to add a Helm repository in the OpenShift Console. We would like to add the ability to install Helm repository as a custom resource but with `scope: namespaced`.
```

Comment:
are there any console, or other tooling integrations with the existing Cluster scoped resource, that will need to be updated to also look at the namespaced resources?

If so, i'd expect those to be called out in this EP and those teams to be tagged as reviewers.

Reply:
I think Console team for sure @zonggen I would invite @christianvogt  and @rohitkrai03 to review

Reply:
Added more details to implementation details section.

Reply:
they should be added as reviewers in the reviewer section w/ a blurb making it clear what aspect of this you want their review of.

Reply:
Yes the console developer catalog would need to be updated to handle namespace scoped helm chart repositories.